source/JavaPropertiesParser.Tests/JavaPropertiesParser.Tests.csproj

```diff
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <TargetFramework>net6.0</TargetFramework>
 
         <IsPackable>false</IsPackable>
     </PropertyGroup>
 
     <ItemGroup>
         <PackageReference Include="Assent" Version="1.6.1" />
-        <PackageReference Include="Fastenshtein" Version="1.0.0.5" />
-        <PackageReference Include="FluentAssertions" Version="5.10.3" />
+        <PackageReference Include="Fastenshtein" Version="1.0.0.8" />

```

Comment:
Version upgrade are not really needed. Included during development. Didn't remove as all tests are still passing with it.